lib/rspec/rails/adapters.rb

```diff
@@ -38,8 +38,7 @@ def self.disable_testunit_autorun
         gem 'minitest' if defined?(Kernel.gem)
         require 'minitest/unit'
         Assertions = MiniTest::Assertions
-      elsif ::Rails::VERSION::STRING >= '3.2.21'
-        # TODO: Change the above check to >= '3.2.22' once it's released
+      elsif ::Rails::VERSION::STRING >= '3.2.22'
```

Comment:
@cupakromer can you remember why is necessary? When is rails going to be higher than this but lower than 4.0.0?